README.md

```diff
@@ -30,14 +42,14 @@ The `libraries` component contains librares developed by the CoDyCo consortium,
 
 The projects downloaded in the `libraries` component are:
 
-* `codyco-commons`: A collection of functions and utilities used in the other projects [Project page](https://github.com/robotology-playground/codyco-commons)
-* `idyntree`: YARP-based Floating Base Robot Dynamics Library [Project Page](https://github.com/robotology/idyntree)
-* `paramHelp`: Library for simplifying the management of the parameters of YARP modules [Project page](https://github.com/robotology-playground/paramHelp)
-* `wholebodyinterface`: C++ Interfaces to sensor measurements, state estimations, kinematic/dynamic model and actuators for a floating base robot [Project Page](https://github.com/robotology-playground/wholebodyinterface)
-* `yarp-wholebodyinterface`: Implementation of the wholeBodyInterface for YARP robots [Project Page](https://github.com/robotology-playground/yarp-wholebodyinterface)
+* [`codyco-commons`](https://github.com/robotology-playground/codyco-commons) : A collection of functions and utilities used in the other projects 
+* [`idyntree`](https://github.com/robotology/idyntree) : YARP-based Floating Base Robot Dynamics Library
+* [`paramHelp`](https://github.com/robotology-playground/paramHelp) : Library for simplifying the management of the parameters of YARP modules
+* [`wholebodyinterface`](https://github.com/robotology-playground/wholebodyinterface) : C++ Interfaces to sensor measurements, state estimations, kinematic/dynamic model and actuators for a floating base robot 
+* [`yarp-wholebodyinterface`](https://github.com/robotology-playground/yarp-wholebodyinterface) : Implementation of the wholeBodyInterface for YARP robots 
 * `orocosBFLBerdy`: Fork of the Bayesian Filtering Library [Orocos-BFL](http://www.orocos.org/bfl). *Early stage. Currently used and tested only for the quaternionEKF module and basic Kalman Filtering*
```

Comment:
This needs to change into wholeBodyEstimator instead of quaterionEKF which I should have removed. If not, I will do it soon.